infra/terraform/global/variables.tf

```diff
@@ -25,3 +25,7 @@ variable "global_cloudtrail_bucket_name" {
 variable "s3_logs_bucket_name" {
   default = "moj-analytics-s3-logs"
 }
+
+variable "helm_repo_s3_bucket_name" {
+  default = "moj-analytics-helm-repo"
```

Comment:
I wonder what created this in the first place? :thinking:

Reply:
💥 🐔 🥚

Reply:
I suspect ✋-lly. 

I wonder if I should actually have a module that:
1. Creates that S3 bucket
2. Creates the IAM user, etc...

I also wonder if we should keep the name of IAM user that can update that S3 bucket/helm repository CI-agnostic and not have "concourse" in it.

Reply:
Fair enough. I'll leave as it is as it's working 👍